wiki-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -1,5 +1,154 @@
 <template>
   <v-app class="notesEditor">
-    <p>This is notes Management portlet!!</p>
+    <div>
+      <div
+        id="notesActivityComposer"
+        class="notesComposer">
+        <div class="notesComposerActions">
+          <div class="notesFormButtons">
+            <div class="notesFormLeftActions mr-10">
+              <img src="/news/images/newsImageDefault.png">
```

Comment:
news?

Reply:
@plamarque this is a draft version because i have problem and i push the code for krout to check with me, i will fix all this once the pr  completed and ready for review :)